### ISSUE TITLE
Improve the Thank you page with a YouTube video

### ISSUE THREAD

**Issue by gary-rowe** [bitcoin-solutions/multibit-website#247]
User feedback indicates that the post-installation process comes as a surprise and would benefit from a YouTube tutorial.

**Comment by gary-rowe** [comment]
Here's a screen shot of the outcome:
<img width="1167" alt="screen shot 2015-08-09 at 16 27 12" src="https://cloud.githubusercontent.com/assets/342133/9155557/a3b42bee-3eb3-11e5-9086-7b5472354c3b.png">

**Some notes about the approach**
A simple embed was not possible due to the player dropping a tracking cookie. The alternative APIs such as iFrame and JavaScript are now deprecated leaving only the `div` mechanism which does not provide any way to switch off the tracking cookie from `doubleclick.net`.  

To overcome this I just did a screenshot of the video and made an offsite link to YouTube. In this way people visiting are not subjected to tracking of downloading MultiBit. There is also the nice side effect that NoScript users are unaffected. Ghostery remains at zero.

Since the video link is small I've added a quick install guide to the right that might get some attention. This approach necessitates a responsive layout but that doesn't allow the guide to wrap under the image without a bit more trickery so I'll leave it for another time.

Ready for review and close.
